lib/auth/testauthority/testauthority.go

```diff
@@ -73,6 +74,43 @@ func (n *Keygen) GenerateUserCert(c services.UserCertParams) ([]byte, error) {
 	return n.GenerateUserCertWithoutValidation(c)
 }
 
+func (n *Keygen) GenerateJWT() ([]byte, []byte, error) {
```

Comment:
Could we get either a comment or a named return to differentiate the byte slices?